dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestGeneratorInterface.cpp

```diff
@@ -80,6 +80,10 @@ TEST(DataInterfaceTest, Generator_1) {
   genItf.setVoltageLevelInterface(vlItf);
   ASSERT_EQ(genItf.getID(), "GEN1");
 
+  ASSERT_FALSE(genItf.hasInitialConditions());
```

Comment:
I'd try as much as possible to test the hasInitialConditions setting P,Q,V,angle (it depends on the component) via myComponent.getTerminal().setBlaBla